LuaSkin/LuaSkin/Skin.m

```diff
@@ -294,6 +338,15 @@ - (void)checkArgs:(int)firstArg, ... {
                 break;
             case LUA_TNUMBER:
                 lsType = LS_TNUMBER;
+
+                if ((spec & lsType) && (spec & LS_TINTEGER)) {
+                    int isInteger ;
+                    lua_tointegerx(_L, idx, &isInteger) ;
+                    if (!isInteger) {
```

Comment:
Not sure I follow... "pass a float"?  Do you mean in the arguments on the Lua stack?  That's the whole point here -- to detect whether or not they've put in something that isn't an integer (or immediately convertible into and integer) or not...

Do you have an example of what you have in mind?